config.py

```diff
@@ -34,3 +34,6 @@
   'Alice': '221b Baker Street, London NW1 6XE',
   'Bob': 'Buckingham Palace, London SW1A 1AA'
 }
+
+house_member_email_addresses = {
```

Comment:
I thought of that, but I don't want to use an email address that actually exists! I guess I can just do example@gmail.com unless you have any other suggestions?